src/App.js

```diff
@@ -1,34 +1,37 @@
 import { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
+import {
+  Navigate, BrowserRouter, Routes, Route,
+} from 'react-router-dom';
 import './App.css';
 import './Style/register.css';
 import './Style/medias.css';
-import './Style/myReservation.css';
-import { getExperiencesData } from './Redux/Slices/ExperiencesSlice';
 import { loadUserFromLocalStorage } from './Redux/Slices/usersSlice';
-import { getUserReservations } from './Redux/Slices/reservationsSlice';
+import LoginPage from './Components/LoginPage';
+import Register from './Components/Register';
+import RoutesWrapper from './Components/RoutesWrapper';
 
 function App() {
   const dispatch = useDispatch();
   const status = useSelector((state) => state.experiences.status);
-  const { id } = useSelector((state) => state.users.user);
 
   useEffect(() => {
     if (status === 'idle') {
       // Set user infromation from localStorage
       dispatch(loadUserFromLocalStorage());
-      dispatch(getExperiencesData());
     }
   }, [dispatch, status]);
 
-  useEffect(() => {
-    if (status === 'succeeded') {
-      dispatch(getUserReservations(id));
-    }
-  });
-
   return (
-    <div className="Routes" />
+    // <div className="Routes" />
+    <BrowserRouter>
+      <Routes>
+        <Route path="/login" element={<LoginPage />} />
+        <Route path="/" element={<Navigate to="/login" />} />
+        <Route path="/Register" element={<Register />} />
+        <Route path="/*" element={<RoutesWrapper />} />
+      </Routes>
```

Comment:
Please check the redirection when the user is `Authenticated`, when I register the page redirects successfully to the `MainPage` but when I `Login` I can't access the `MainPage` and I continue in the `Login` page.